dask_expr/io/parquet.py

```diff
@@ -102,6 +103,13 @@ def columns(self):
             return pd.Index(_list_columns(columns_operand))
 
     def _simplify_up(self, parent):
+        if isinstance(parent, Index):
+            # Column projection
+            operands = list(self.operands)
+            operands[self._parameters.index("columns")] = []
+            operands[self._parameters.index("_series")] = False
+            return ReadParquet(*operands)
```

Comment:
This feels similar to a stalled experiment of mine in dask/dask: https://github.com/dask/dask/pull/10072#issuecomment-1470962862

In that PR, I found that making an empty column projection here can be dangerous.

Reply:
I like the optimization, and have no immediate concerns. I'm just pointing this out in case we end up seeing strange `len`/`index` behavior.

Reply:
Grand.  Thanks!